dotnetapp-buildcontainer/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
For both script files, what are your thoughts on handling error cases.  I am primarily thinking about the experience when there is a build error in the app that is getting built.

This really comes down to how complete of a sample are we trying to build and what are we trying to showcase?  Is this sample something we would want devs to be able to copy into their workflow with relatively few changes? Do we want to keep this sample relatively simple in order to better showcase the flow that can be achieved?